bookmark.h

```diff
@@ -0,0 +1,56 @@
+#ifndef BOOKMARK_H
+#define BOOKMARK_H
+#include <string>
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+class Bookmark {
+        public:
+            void displayMainMenu();
+            void addUser(int userCounter);
+            void editUser(int userCounter);
+            void viewAllUsers(int userCounter);
+            void deleteUser(int userCounter);
+
+            void addBook(int bookCounter);
+            void deleteBook(int bookCounter);
+            void editBook(int bookCounter);
+            void searchBook(int bookCounter);
+            void viewAllBooks(int bookCounter);
+            void displayTopBooks();
+
+
+//counter for book array and user array
+    int bookCounter = 0;
+    int userCounter=0;
+
+//function to increase counter
+    void increment(int a) {
```

Comment:
parameter naming is bad in this case. 
❌ never use naming like `a`, `x` or similar.
The parameter should be called `newCounter`.
Also, I am not sure that you need an additional parameter in those functions. Since the method name is called increment it can be called without param.

So this can look like:

`void increment() {
bookCounter ++;
}`